src/Bicep.Core/bicepsettings.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Linter": {

```

Comment:
>Linter [](http://example.com/codeflow?start=3&length=6)

What went into choosing a custom JSON format for the config vs choosing something like .editorconfig?

Reply:
Discussion Summary: json settings seem to be more in line with the target ecosystem than something like .editorconfig since the settings will apply in both the "editor" and for CLI. General impression is that .editorconfig is more associated with Visual Studio than VsCode.